packages/twenty-front/src/modules/settings/data-model/object-details/components/SettingsObjectItemTableRow.tsx

```diff
@@ -34,12 +35,18 @@ export const SettingsObjectItemTableRow = ({
   action,
   objectItem,
   to,
+  updateInstanceCount,
 }: SettingsObjectItemTableRowProps) => {
   const theme = useTheme();
 
   const { totalCount } = useFindManyRecords({
     objectNameSingular: objectItem.nameSingular,
   });
+  useEffect(() => {
```

Comment:
Can you try getting rid of useEffect? https://react.dev/learn/you-might-not-need-an-effect

Reply:
Let me find a way to do that. Probably I will have to create a global state to do that, as we have to store the data somewhere to actually use it higher up the React tree.

Reply:
You can use a recoil family here also